oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGCSupport.java

```diff
@@ -19,12 +19,28 @@
 
 package org.apache.jackrabbit.oak.plugins.document;
 
+import static java.util.Comparator.comparing;
+import static java.util.Optional.empty;
+import static java.util.Optional.of;
+import static java.util.Optional.ofNullable;
+import static java.util.stream.Stream.concat;
+import static java.util.stream.StreamSupport.stream;
 import static org.apache.jackrabbit.guava.common.collect.Iterables.filter;
+import static java.util.stream.Collectors.toList;
+import static org.apache.jackrabbit.guava.common.collect.Sets.newHashSet;
```

Comment:
(avoid use of Guava in new code)

Reply:
removed guava usage in https://github.com/apache/jackrabbit-oak/pull/1459